docs/user_manual/details.md

```diff
@@ -12,7 +12,7 @@ Columns containing QC metrics provide tool tips detailing the exact required thr
 the recorded number to see its requirements. The cell will be shaded red if the requirement is not
 met, and yellow if the value is unavailable.
 
-![Metric cell](/images/metric_cell.png)
+![Metric cell](../images/metric_cell.png)
```

Comment:
The changes below point to `../../images/`; does this need updating?

Reply:
Both worked, but it was weird. I intended to make all links relative based on the deployed site, but it's better to make them relative to the source files and let mkdocs do its thing to update them during build. Updated all links